### DISCUSSION TITLE
Google leaks say we're winning!

### DISCUSSION THREAD

**Discussion by Alumniminium** [ggml-org/llama.cpp#1339] [General]
https://natural20.com/google-ai-documents-leak/

> LLMs on a Phone: People are running foundation models on a Pixel 6 at 5 tokens / sec.
> While our models still hold a slight edge in terms of quality, the gap is closing astonishingly quickly.
> Open-source models are faster, more customizable, more private, and pound-for-pound more capable.
> They are doing things with $100 and 13B params that we struggle with at $10M and 540B.

> March 18, 2023 – Now It’s Fast
> Georgi Gerganov uses 4 bit quantization to run LLaMA on a MacBook CPU. It is the first “no GPU” solution that is fast enough to be practical.

Congrats @ggerganov

**Comment by FNsi** [comment]
Need Merge more and more Loras
